### PR TITLE
Add `Mapper's Choice Awards / A Year In Mapping 2023` news post

### PR DESCRIPTION
**Pull request by Hiviexd**

### CODE REVIEW

File: news/2024/2024-02-01-mappers-choice-awards-2023.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Mapper's Choice Awards / A Year In Mapping 2023
+date: 2023-02-01 00:00:00 +0000
```
Comment (Walavouchey) [inline news/2024/2024-02-01-mappers-choice-awards-2023.md:4]:
```suggestion
date: 2024-02-01 00:00:00 +0000
```

File: news/2024/2024-02-01-mappers-choice-awards-2023.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+[Just like last year](https://osu.ppy.sh/home/news/2023-03-03-mappers-choice-awards), there are many awards for the winners and runner-ups of every category! If you'd like to learn more, keep reading here and check out the [Corsace website](https://corsace.io/), where you can find the results of previous iterations of MCA and learn more about these events!
+
+In addition, A Year In Mapping is what one would call the *Spotify Wrapped* for the osu! mapping and modding community, showcasing a variety of intriguing statistics gathered from across last year's maps, such as total number of guest difficulties, most mapsets ranked, most favourites and more. If this sort of thing interests you, read on!
+
+## Mapper's Choice Awards 2023
+
+If you ranked any beatmap, guest difficulty, or storyboard in 2023, you're eligible to help the mapping community pick the most standout maps of the year!
+
+Just like the previous editions of MCA, there are three stages: nomination, voting, and results. See below to know when the deadlines for nomination and voting are, and when to expect the final results!
```
Comment (Walavouchey) [inline news/2024/2024-02-01-mappers-choice-awards-2023.md:21]:
```suggestion
Just like the previous editions of MCA, there are three stages: nomination, voting, and results. [See below](#schedule) to know when the deadlines for nomination and voting are, and when to expect the final results!
```

File: news/2024/2024-02-01-mappers-choice-awards-2023.md (hunk 1 above)
Comment (Hiviexd) [reply news/2024/2024-02-01-mappers-choice-awards-2023.md:4]: 2nd time this year...

File: news/2024/2024-02-01-mappers-choice-awards-2023.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+- **Nomination:** Eligible users can submit any beatmaps, storyboards, or users from 2023 which they feel are deserving of recognition in their respective categories.
+- **Voting:** Eligible users can vote on the pool of submitted nominations.
+- **Results:** Will be based off of the results of voting and feature a livestream announcing each podium of every category.
```
Comment (Walavouchey) [inline news/2024/2024-02-01-mappers-choice-awards-2023.md:25]:
```suggestion
- **Results:** These will be based on the voting tallies and feature a livestream announcing the podium of each category.
```

File: news/2024/2024-02-01-mappers-choice-awards-2023.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  - Category winners: 2 months per category
+- **Badges**
+  - Winners of the Grand categories will be awarded special badges each.
+  - The winner of each regular category will be awarded a more generic badge.
+
+## A Year In Mapping 2023
+
+A ton of data and stats around the mapping trends of 2023 are now available for you to see in A Year In Mapping (*AYIM*) 2023!
+
+Per usual, A Year in Mapping features all the coolest statistics of the year all in [one place](https://ayim.corsace.io/2023/). Want to know who ranked the hardest or easiest map this year? Who ranked the most difficulties? Look no further than AYIM!
```
Comment (Walavouchey) [inline news/2024/2024-02-01-mappers-choice-awards-2023.md:53]: a big link on one line would be better

File: news/2024/2024-02-01-mappers-choice-awards-2023.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+---
+
+Now that you're all caught up on the schedule and the events, it's time to get to the important part: nominating and voting! Head on over to the [website](https://mca.corsace.io/2023/) to get started!
```
Comment (Walavouchey) [inline news/2024/2024-02-01-mappers-choice-awards-2023.md:57]: here too

**Review by Walavouchey — CHANGES_REQUESTED** [review]
bold **mappers' choice awards** and **a year in mapping**

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**